drydock/plugin.py

```diff
@@ -135,7 +135,6 @@ def get_sync_waves_for_resource(resource_name: str) -> SYNC_WAVES_ORDER_ATTRS_TY
         "INGRESS": False,
         "INGRESS_EXTRA_HOSTS": [],
         "INGRESS_LMS_EXTRA_HOSTS": [],
-        "NEWRELIC": False,
```

Comment:
Why do you removed this setting?

Reply:
It was an old setting that is no longer used. We enable the newrelic patches by setting `DRYDOCK_NEWRELIC_LICENSE_KEY` to a non empty value.